src/main/java/org/parabot/core/desc/ServerProviderInfo.java

```diff
@@ -141,4 +151,17 @@ public Properties getProperties() {
     public HashMap<String, Integer> getSettings() {
         return settings;
     }
+
+    public URL getRandoms() {
+        try {
+            String randomsUrl = properties.getProperty("randoms_jar");
+            if (randomsUrl == null || randomsUrl.length() == 0) {
+                randomsUrl = Configuration.GET_RANDOMS + (Configuration.BOT_VERSION.isNightly() ? Configuration.NIGHTLY_APPEND : "");
+            }
+            return new URL(randomsUrl);
```

Comment:
Shouldn't this be **within** the if-statement, so it returns null instead?

Reply:
No, if the initial `properties.getProperty("randoms_jar");` is null then it'll fall back to the default BDN URL. I'll add a comment clarifying this.